server.js

```diff
@@ -257,6 +257,12 @@ app.use('/pl/administrator/overview', require('./pages/administratorOverview/adm
 //////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////
 //////////////////////////////////////////////////////////////////////
+// Webhooks //////////////////////////////////////////////////////////
+app.use('/pl/webhooks/autograder', require('./webhooks/autograder/autograder'));
```

Comment:
A less "hacky" solution could be to write a wrapper around `authn.js` that takes a list of routes to not do auth on; functionality would be the same as modifying `authn.js` but might adhere to separations of concern a bit better.

Thanks for pointing out that it won't work as is though, that probably saved me a lot of debugging time in prod!